srcpkgs/gamescope/patches/0002-use-system-SPIRV-Headers.patch

```diff
@@ -0,0 +1,25 @@
+From 1dd1cc1fd6ee0feb7f1caf34873e67ed8568c650 Mon Sep 17 00:00:00 2001
+From: Dexter Gaon-Shatford <dexter@gaonshatford.ca>
+Date: Sun, 12 Nov 2023 10:24:22 -0500
+Subject: [PATCH] use system SPIRV-Headers
+
+---
+ src/meson.build | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/meson.build b/src/meson.build
+index 60d6b5a..db54651 100644
+--- a/src/meson.build
++++ b/src/meson.build
+@@ -94,7 +94,7 @@ reshade_src = [
+ reshade_include = include_directories([
+   'reshade/source',
+   'reshade/include',
+-  '../thirdparty/SPIRV-Headers/include/spirv/unified1'
++  '/usr/include/spirv/unified1'
```

Comment:
this should be relative to XBPS_CROSS_BASE, so maybe a vsed would be better (would probably remove the need for spirv-headers on host)